src/web/Keycloak.ts

```diff
@@ -1,5 +1,20 @@
 import Keycloak from 'keycloak-js';
 
-const keycloak = new Keycloak('/api/keycloak-config');
+let keycloakInstance: Keycloak | null = null;
 
-export default keycloak;
+export const getKeycloakInstance = () => {
+  if (!keycloakInstance) {
+    throw new Error(
+      'No Keycloak instance found. Please call createKeycloakInstance() to initialize one.'
```

Comment:
Would it be bad to create the instance here if it doesn't already exist?

Reply:
I think I prefer it as-is because we only ever want one keycloak instance in production, which I'm thinking should just be creating by calling `createKeycloakInstance()`, which should only be called once. I think that makes it easier to ensure only one is ever created. Does that some reasonable?